doc/writing-puzzles.md

```diff
@@ -61,11 +58,9 @@ unique: you may not have two 5-point puzzles.
 Point values should roughly reflect how difficult a problem is to solve.
 It's not terribly important that a 200-point puzzle be ten times harder
 than a 20-point puzzle, but it is crucial that a 25-point puzzle be
-roughly as difficult as a 20-point puzzle.  Poorly-weighted puzzles has
+roughly as difficult as a 20-point puzzle.  Poorly-weighted puzzles have
```

Comment:
subject/object agreement! "X has been the main reason" or "X have been the main reason"? In this case "poorly-weighted puzzles" is a single condition. If I'd written "Puzzles that are poorly-weighted" then "have" would be correct.

#EnglishPedantry